apps/etl-func/docs/tokenize.puml

```diff
@@ -0,0 +1,29 @@
+@startuml TokenizeFiscalCode
+
+participant caller
+participant TokenizeFiscalCode as tf
+participant PDV
+participant "ai-common" as ai
```

Comment:
are we sure that we want to log errors on AI?

Reply:
maybe it's better to just return an error and leave the log to the caller